src/components/InputItem/Input.module.css

```diff
@@ -1,9 +1,57 @@
-.addtask__wrapper {
+.input_wrap {
     display: flex;
-    flex-flow: column nowrap;
-    align-items: center;
+    flex-flow: row nowrap;
+    margin: auto;
+    position: absolute;
+    bottom: 30px;
+    justify-content: center;
+    width: 700px;
 }
 
 .input {
-    width: 250px;
-}
\ No newline at end of file
+    width: 592px;
+    height: 36px;
+    background: #FFFFFF;
+    border: 1px solid #CCCCCC;
+    box-sizing: border-box;
+    border-radius: 29px;
+    outline: none;
+    padding-left: 10px;
+}
+
+.btn_input {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)